controller/input.cpp

```diff
@@ -191,7 +201,7 @@ void InputDevice::handleFeedbackErase(uint32_t id)
         return;
     }
 
-    effect = {};
+    effects[erase.effect_id] = {};
```

Comment:
Same here, bounds checking is missing. Maybe create accessor function instead of accessing the array directly, that is: make it private, create functions `setEffect` and `getEffect`.

Reply:
Using accessor function will make it easier later to change the implementation to maps if someone wants to. But I think a static array is perfectly okay, ff-memless does probably something similar: It also has a limited amount of memory. We can just skip the overhead from map lookups, a static array should be okay. Still using accessor function allows for limiting changes to code to one central place.